data/mods/Aftershock/README.md

```diff
@@ -13,18 +14,30 @@ Please feel free to reach out to us about ideas and implementations.
 
 These are the files for Aftershock. Whatever you're looking for has been sorted into subfolders for ease of access:
 
+### itemsgroups
+
+Contains json data for itemgroups.
+
 ### items
 
-Contains json data for food, weapons, tools, and materials.
+Contains json data for all item types.
 
 ### maps
 
-Contains json data for map generation, item groups, and furniture.
+Contains json data for map generation, chunks, terrain, map specials and furniture.
 
 ### mobs
 
 Contains json data for new monsters and monster spawn groups.
 
+### monsterdrops
+
+Contains the json which controls what monsters drop on death.
+
+### muations
+
+Contains json for mutations and threshold dreams.
+
 ### player
 
 Contains json data for things that affect the player; bionics, mutations, professions, techniques, status effects, etc.
```

Comment:
```suggestion
Contains json data for things that affect the player; bionics, professions, techniques, status effects, etc.
```
Bionics are not in items?

Reply:
Bionics are strange; acting as both trait-like things, and an item.

I imagine there will be some bionics that will only be accessible via professions, and can't be installed via normal game play

Reply:
Even then they'd have to be items as well because otherwise it would cause error's when you remove the bionic.